builtin/aws/ecs/platform.go

```diff
@@ -708,10 +711,17 @@ func createALB(
 		} else {
 			s.Update("Creating new ALB: %s", lbName)
 
+			scheme := "internet-facing"
```

Comment:
nit: could this be
  
         scheme := elbv2.LoadBalancerSchemeEnumInternetFacing

and then below


              scheme = elbv2.LoadBalancerSchemeEnumInternal

to future proof if AWS changes the strings in the future?

Reply:
Good call, forgot about those.